carbon-design-system/carbon-for-ibm-dotcom
# fix(tab-extended): remove tab content area padding + update demo content

### Related Ticket(s)

DDS Consulting Issue: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/7206
HC JIRA Ticket: https://jsw.ibm.com/browse/HC-2247

### Description
- This PR removes the padding value that's set in the `.bx--tab-content` class and updates the Storybook demo to showcase content block content for layout alignment.

### Testing Instructions

- Visit the `Tab Extended` layout Component - `/?path=/story/components-tabs-extended--default`.
- Ensure that there's no padding within the Tab Content class.
